src/twisted/python/test/strategies.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
+
+"""
+Hypothesis strategies for values related to L{twisted.python}.
+"""
+
+from hypothesis.strategies import SearchStrategy, characters, text
+
+
+def systemd_descriptor_names() -> SearchStrategy[str]:
```

Comment:
My thinking here is definitely along the lines of improving re-use.  Having a convenient strategies modules with all kinds of fun stuff in it seems like it might encourage that.  I don't really know.  I haven't worked with any large teams on projects that use Hypothesis so I've only tried these strategies on myself before. :slightly_smiling_face: 

I do know notice that this strategy name violates the Twisted naming convention though...  Hard to keep straight these days :/

Reply:
A little bit more on this...  Perhaps this systemd strategy will only ever be used by `test_systemd`.  I still think it's a good idea to have it out of the test module itself.  The test module's job is not to supply the canonical definition of a systemd "FileDescriptorName" value.  The test module's job is to define some behavior _in terms of_ that.  If I could import this systemd-related strategy from a third-party "systemd" package then I think that would be even better.  Since there is no such thing, I think putting the strategy in a general "strategies" module in Twisted is the next best thing.